### DISCUSSION TITLE
Stabillity in online server using Vultr and n8n

### DISCUSSION THREAD

**Discussion by 2tmmkt** [devlikeapro/waha#626] [Q&A]
I want to automatize some processes in my clients WhatsApp sales. I have managed to create a Docker container in My Vultr server and connected it to my n8n, but the big problem is that the connection between the cell phone and the swagger falls down all the time and don't keep it enough time for me tu run recorrent processes.

Could you guys help me with that? Someone uses this dynamics with Vultr and n8n?

For you to analyse better, my server configs in Vultr are: 1Gb of RAM, 1 vCPU and 1 Bandwith

Should I pay a server with better configs? Or this is not a server config problem but a container config problem? 

I can provide more details if it is needed!

**Comment by devlikepro** [comment]
Hi! 
1. Did you follow the guide to install docker container? https://waha.devlike.pro/docs/how-to/install/
2. Could you check if there's any errors in docker logs?

>  that the connection between the cell phone and the swagger falls down all the time

3. How often? May be after what time it happens?

> Vultr

Do you have Ubuntu VPS there, right? 




[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
>1Gb of RAM, 1 vCPU and 1 Bandwith

👉 We kindly recommend getting **at least 2CPU** and **2GB RAM** for a VPS, it’s not comfortable to work with less resources.

https://waha.devlike.pro/docs/overview/faq/#how-much-cpu-and-memory-does-waha-consume



[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by 2tmmkt** [comment]
Thank you guys for your support. The main problem was the server config. I just upgraded its RAM memory and everything worked!
